src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx

```diff
@@ -7,6 +7,49 @@ topics:
 
 Use this document to learn how to review and peer edit docs for you fellow writers in GitHub. Check the [Tech Writer workflow doc](/docs/style-guide/writer-workflow/tech-writer-workflow/) for info on how to set up your local environment. To troubleshoot GitHub issues, see [our guide](/docs/style-guide/writer-workflow/github-troubleshooting/).
 
+## What needs a peer review [#peer-reqs]
+
+Any substantial pull request should be peer reviewed. The line between substantial and trivial is a bit blurry, so refer to the following table when in doubt:
```

Comment:
Team meeting notes recommend any convos about review should take place in the PR itself as much as possible. IIRC we also agreed that sometimes convos need to take place in slack, so try to summarize that convo for public consumption in the PR. Recommend that you add something about this to style guide.